ruby/vampires.rb

```diff
@@ -0,0 +1,68 @@
+puts "How many employee applications will you be processing today?"
+number_of_applications = gets.chomp.to_i
+
+number_of_attempts = 0
+
+while
+  number_of_attempts < number_of_applications
+
+puts "What is your name?"
+  name = gets.chomp.capitalize
+  
+puts "How old are you?"
+  age = gets.chomp.to_i
+  
+puts "Really, what year were you born?"
+  birth_year = gets.chomp.to_i
+  
+puts "Would you like some garlic bread for lunch today? (yes or no)"
+  garlic = gets.chomp
+  
+puts "Will you be enrolling in our health insurance policy? (yes or no)"
+  insurance = gets.chomp
+
+#just to double check quickly if age is accurate (it is known that it is not visible in the program but referenced later on in the qualifiers)
+age_year_match = 2017 - birth_year
+
+allergies = " "
+
+while 
```

Comment:
Take a look at looping syntax again.  Why is the problematic?

What is happening here that could cause unexpected behavior?